controllers/core/event_controller.go

```diff
@@ -47,7 +49,20 @@ const (
 	ReconcileEventsLatencyMetrics = "reconcile_events"
 	LabelOperation                = "operation"
 	LabelTime                     = "time"
+	LabelCacheHit                 = "cache_hit"
+	LabelCacheMiss                = "cache_miss"
+	LabelCacheSetErr              = "cache_set_error"
+	CacheHitMetricsValue          = "NODE_CACHED"
+	CacheMissMetricsValue         = "NODE_NOT_CACHED"
+	CacheErrMetricsValue          = "NODE_CACHE_ERROR"
 	EventRegardingKind            = "Node"
+	SGPPrefix                     = "SGP"
+	CNPrefix                      = "CN"
+
+	// customized configurations for BigCache
+	InstancesCacheTTL     = 30 * time.Minute // scaling < 1k nodes should be under 20 minutes
```

Comment:
As discussed offline, lets check how to delete the cache entry when the node is getting deleted (maybe node controller can do it). Since for 30 mins we will have stale data and node churn is a very common scenario.

Reply:
Th new updated commit has the change. Thanks.